shims/spark300/src/main/scala/com/nvidia/spark/rapids/shims/spark300/GpuHashJoin.scala

```diff
@@ -199,20 +200,24 @@ trait GpuHashJoin extends GpuExec with HashJoin {
       }
 
       override def hasNext: Boolean = {
-        while (nextCb.isEmpty && (first || stream.hasNext)) {
+        var may_continue = true
```

Comment:
nit: `may_continue` does not follow the naming conventions we have.  Please use `mayContinue` instead.

Reply:
done